client/reader/stream/index.jsx

```diff
@@ -470,9 +409,13 @@ class ReaderStream extends React.Component {
 			);
 			showingStream = true;
 		}
+		const indexOfColon = streamKey.indexOf( ':' );
+		const streamType = indexOfColon === -1 ? streamKey : streamKey.substring( 0, indexOfColon );
+
 		const TopLevel = this.props.isMain ? ReaderMain : 'div';
 		return (
 			<TopLevel className={ classnames( 'following', this.props.className ) }>
+				{ ! streamType === 'search' && <Interval onTick={ this.poll } period={ EVERY_MINUTE } /> }
```

Comment:
`streamType !== 'search'` would be simpler i think

Reply:
agreed! thanks i'll update